Makefile

```diff
@@ -757,6 +776,9 @@ endif
 	@printf "\tKnown: $(sort $(HYPERVISORS))\n"
 	@printf "\tAvailable for this architecture: $(sort $(KNOWN_HYPERVISORS))\n"
 	@printf "\n"
+	@printf "• Features:\n"
+	@printf "\tSELinux (FEATURE_SELINUX): $(FEATURE_SELINUX)\n"
```

Comment:
maybe you want to replace one of these `FEATURE_SELINUX` with `SELINUXTAG` ... ?

Reply:
`FEATURE_SELINUX` is the user-visible bit that can be tweaked at build time while `SELINUXTAG` is just an implementation detail, so I don't think it makes sense to display the latter.